github-actions/lock-closed/action.yml

```diff
@@ -5,6 +5,9 @@ inputs:
   github-token:
     description: 'Github access token'
     required: true
+  max-per-execution:
```

Comment:
Can we change this to `locks-per-execution` and then enforce a max in the code?

Reply:
Done.